src/components/application_manager/include/application_manager/policies/policy_handler.h

```diff
@@ -183,9 +183,7 @@ class PolicyHandler : public PolicyHandlerInterface,
                          const std::string& policy_app_id,
                          const std::string& hmi_level) OVERRIDE;
 
-#ifndef EXTERNAL_PROPRIETARY_MODE
-  void OnPTUTimeOut() OVERRIDE;
-#endif
+  void OnPTUTimeOut();
```

Comment:
@ydementieiev OVERRIDE?

Reply:
@AKalinich-Luxoft   `EXTERNAL_PROPRIETARY_MODE` build failed with `OVERRIDE`

Reply:
@ydementieiev then add pure virtual `OnPTUTimeOut` into interface class for `EXTERNAL_PROPRIETARY_MODE`